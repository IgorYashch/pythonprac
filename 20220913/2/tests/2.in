[17, -2, 57, 34, -10, 1.0]
