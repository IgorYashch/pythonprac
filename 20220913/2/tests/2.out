-10, -2, 1.0, 17, 34, 57
